src/libs/actions/Transaction.ts

```diff
@@ -86,6 +87,12 @@ function saveWaypoint(transactionID: string, index: string, waypoint: RecentWayp
     if (!lodashHas(waypoint, 'lat') || !lodashHas(waypoint, 'lng')) {
         return;
     }
+
+    // If current location is used, we would want to avoid saving it as a recent waypoint. This prevents the 'Your Location'
+    // text from showing up in the address search suggestions
+    if (isEqual(waypoint?.address, CONST.YOUR_LOCATION_TEXT)) {
```

Comment:
NAB: Why is lodash used here? Wouldn't `===` work just as well? It's only comparing two strings.

Reply:
Also, don't we have an eslint rule for TS that says we prefer _not_ to use lodash or underscore?